app/cafe.py

```diff
@@ -0,0 +1,20 @@
+import datetime
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("All friends should be vaccinated")
+        if visitor["vaccine"]["expiration_date"] < datetime.date.today():
+            raise OutdatedVaccineError("Your vaccine is out of date")
+        if not visitor["wearing_a_mask"]:
+            raise NotWearingMaskError("You should wear mask")
+        return f"Welcome to {self.name}"

```

Comment:
The function `visit_cafe` could benefit from a docstring explaining what it does, what parameters it expects, and what it returns. This would enhance the readability and maintainability of your code.